internal/app/app.go

```diff
@@ -178,3 +200,80 @@ func loadChains(cfg config.NeutronQueryRelayerConfig, logRegistry *nlogger.Regis
 
 	return neutronChain, targetChain, nil
 }
+
+type connectionParams struct {
+	neutronChainID  string
+	neutronClientID string
+
+	targetChainID      string
+	targetClientID     string
+	targetConnectionID string
+}
+
+func loadConnParams(ctx context.Context, neutronClient, targetClient *rpcclienthttp.HTTP, restAddress string, neutronConnectionId string, logger *zap.Logger) (*connectionParams, error) {
```

Comment:
we have here **neutron**Client and **target**Client, but after them a simple `restAddress` which is ambiguous for it's not said which chain is meant